WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductRowViewModel.swift

```diff
@@ -400,5 +514,25 @@ private extension ProductRowViewModel {
                                                        comment: "Label for one product variation when showing details about a variable product")
         static let pluralVariations = NSLocalizedString("%ld variations",
                                                         comment: "Label for multiple product variations when showing details about a variable product")
+        static let formattedProductSubscriptionBilling = NSLocalizedString(
+            "ProductRowViewModel.formattedProductSubscriptionBilling",
+            value: "%1$@ / %2$@ %3$@",
+            comment: "Description of the subscription price for a product, with price and billing frequency. " +
+            "Reads as: '$60.00 / 2 months'.")
+        static let formattedProductSubscriptionConditions = NSLocalizedString(
+            "ProductRowViewModel.formattedProductSubscriptionConditions",
+            value: "%1$@ signup · %2$@ %3$@ free",
+            comment: "Description of the subscription conditions for a subscription product, with signup fees and free trials." +
+            "Reads as: '$25.00 signup · 1 month free'.")
+        static let formattedProductSubscriptionConditionsWithoutSignup = NSLocalizedString(
+            "ProductRowViewModel.formattedProductSubscriptionConditionsWithoutSignup",
+            value: "%1$@ %2$@ free",
+            comment: "Description of the subscription conditions for a subscription product, with only free trial." +
+            "Reads as: '1 month free'.")
+        static let formattedProductSubscriptionConditionsWithoutTrial = NSLocalizedString(
+            "ProductRowViewModel.formattedProductSubscriptionConditionsWithoutTrial",
+            value: "%1$@ signup",
+            comment: "Description of the subscription conditions for a subscription product, with signup fees but no trial." +
+            "Reads as: '$25.00 signup'.")
```

Comment:
Suggestion: you can put these strings in a sub enum so that the names can be shorter:

```Swift
enum Subscription {
    static let formattedBilling = ""
    static let formattedConditions = ""
    static let formattedConditionsWithoutSignupFee = ""
    static let formattedConditionsWithoutTrial = ""
}
```

Reply:
Also, for the signup fee, should we write "signup fee" instead of just "signup" for clarity? I'm not sure if the wording was decided by a designer, so please take this with a grain of salt.

Reply:
Neat! Changed on f9f3428a5eb238abadd234767097556e986084ae

>  for the signup fee, should we write "signup fee" instead of just "signup" for clarity? I'm not sure if the wording was decided by a designer, so please take this with a grain of salt.

It comes from design indeed! Left it as "signup"